modules/sdk-coin-apt/src/lib/iface.ts

```diff
@@ -15,7 +15,7 @@ export interface AptTransactionExplanation extends BaseTransactionExplanation {
 export interface TxData {
   id: string;
   sender: string;
-  recipient: TransactionRecipient;
+  recipients: TransactionRecipient[];
```

Comment:
this is a breaking change, in order to keep the code backward compatible don't modify the publicly exposed fields and methods. (Open/Closed principle i.e. open for extension, closed for modification).